### ISSUE TITLE
Can‘t build success with maven.

### ISSUE THREAD

**Issue by ghost-cai** [soot-oss/SootUp#805]
windows11
java11
IDEA
sootup1.2

I try to use the sootup, but can' build it.
I try to ues it as maven+pom.xml or as local project modules.
They are all can't build success. 
![image](https://github.com/soot-oss/SootUp/assets/73644390/60df73fb-527f-4d6c-9058-651d1fe7df36)

![image](https://github.com/soot-oss/SootUp/assets/73644390/f6465e14-cea2-48a5-8114-5cf3b8517706)
![image](https://github.com/soot-oss/SootUp/assets/73644390/ea3864d6-0834-48e3-a498-fe8ae7d2179a)
Even I download the com.android.tools:r8:8.0.40.jar in my maven_repository by hand.  It still  reported  `Could not find artifact com.android.tools:r8:pom:8.0.40` and `com.github.ThexXTURBOXx.dex2jar`.
![image](https://github.com/soot-oss/SootUp/assets/73644390/6bd58dea-fcb1-41c3-a850-4185e2aacd22)
Can you give me some suggestions?

**Comment by ghost-cai** [comment]
I solved the problem with `com.github.ThexXTURBOXx.dex2jar`.
I take the 
```
<repositories>
    <repository>
      <id>jitpack.io</id>
      <url>https://jitpack.io</url>
    </repository>
  </repositories>
```
into my pom.xml.
But It still reported `Could not find artifact com.android.tools:r8:pom:8.0.40 in jitpack.io (https://jitpack.io)`.
![image](https://github.com/soot-oss/SootUp/assets/73644390/46c7ef9c-ce41-4493-8541-44667fbac50a)
I have downloaded the `com.android.tools:r8:8.0.40.jar` in my maven_repository by hand.
